causalpy/pymc_models.py

```diff
@@ -340,7 +340,7 @@ def build_model(self, X, Z, y, t, coords, priors):
                 sigma=priors["sigmas"][1],
                 dims="covariates",
             )
-            sd_dist = pm.HalfCauchy.dist(beta=priors["lkj_sd"], shape=2)
+            sd_dist = pm.Exponential.dist(priors["lkj_sd"], shape=2)
```

Comment:
Does this change influence the results in the existing IV notebook?

Reply:
It doesn't change anything really significantly. I did check